container/README.md

```diff
@@ -1,30 +1,237 @@
+# Containers
 
 This document describes how to utilize the provided Dockerfiles for containerization.
 
 ## Dockerfile Selection
 
 ### Freyja Apps
 
-To containerize the [Example Freyja Apps](../freyja_apps/), use
-[Dockerfile.freyja_apps](../Dockerfile.freyja_apps). This dockerfile defaults to the
+To containerize the [Example Freyja Apps](../freyja_apps/) use one of the following dockerfiles:
+
+- [Dockerfile.freyja_apps.amd64](../Dockerfile.freyja_apps.amd64) - For x86-64 architecture.
+- [Dockerfile.freyja_apps.arm64](../Dockerfile.freyja_apps.arm64) - For aarch64 architecture.
+
+These dockerfiles default to the
 [In Memory Example Application](../freyja_apps/in_memory/).
 
-Currently the following example applications can be containerized:
+Currently the following example applications are supported:
 
-- [In Memory Example Application](../freyja_apps/in_memory/)
-- [Ibeji Adapter Example Application](../freyja_apps/ibeji_adapter)
+- [In Memory Example Application](../freyja_apps/in_memory/README.md)
+- [Ibeji Adapter Example Application](../freyja_apps/ibeji_adapter/README.md)
+- [End to End Example Application](../freyja_apps/e2e/README.md)
 
-Each supported application has a README describing the steps to containerize the application.
+See [Docker Containers](#docker-containers) or [Podman Containers](#podman-containers) for
+information on how to build and run supported applications in a container.
 
 ### Freyja Cloud Connectors
 
-Coming soon!
+To containerize the
+[Azure Digitals Twin Cloud Connector](../cloud_connectors/azure/digital_twins_connector/README.md)
+use one of the following dockerfiles:
+
+- [cloud_connectors/azure/Dockerfile.dt_connector.amd64](../cloud_connectors/azure/Dockerfile.dt_connector.amd64) -
+For x86-64 architecture.
+- [cloud_connectors/azure/Dockerfile.dt_connector.arm64](../cloud_connectors/azure/Dockerfile.dt_connector.arm64) -
+For aarch64 architecture.
+
+See [Docker Containers](#docker-containers) or [Podman Containers](#podman-containers) for
+information on how to build and run supported applications in a container.
+
+## Configuration Overrides
+
+This repository provides overrides for the samples listed under
+[Supported Ibeji Samples](../README.md#supported-ibeji-samples). Configuration is provided in the
+[`.freyja`](../.freyja/) directory, with each sample having its own directory. For example, the
+configuration for the
+[Ibeji mixed sample](https://github.com/eclipse-ibeji/ibeji/tree/main/samples/mixed) is provided
+under the [.freyja/mixed_sample](../.freyja/mixed_sample/) directory.
+
+## Docker Containers
+
+### Prequisites
+
+[Install Docker](https://docs.docker.com/engine/install/)
+
+### Running in Docker
+
+To run the service in a Docker container:
+
+1. Run the following command in the project root directory to build the docker container from the
+Dockerfile:
+
+    ```shell
+    docker build -t <image_name> -f <Dockerfile> (optional: --build-arg=APP_NAME=<project name>) .
```

Comment:
You should mention replacing all placeholders with the relevant value before executing/running the  command,.

Reply:
Added note at top of section for docker and podman